tasks.py

```diff
@@ -40,7 +40,24 @@ def tests(c):
     """Run the redis-py test suite against the current python,
     with and without hiredis.
     """
-    run("tox -e plain -e hiredis")
+    print("Starting Redis tests")
+    run("tox -e '{redis,cluster}'-'{plain,hiredis}'")
+
+
+@task
+def redis_tests(c):
```

Comment:
This change means you'll need to update the CONTRIBUTING.md as well.

Reply:
updated